uber/decorators.py

```diff
@@ -337,16 +355,24 @@ def with_restrictions(*args, **kwargs):
     return with_restrictions
 
 
+def set_renderable(func, acccess):
+    """
+    Return a function that is flagged correctly and is ready to be called by cherrypy as a request
+    """
+    func.restricted = getattr(func, 'restricted', acccess)
+    new_func = timed(cached_page(sessionized(restricted(renderable(func)))))
+    new_func.exposed = True
+    return new_func
+
```

Comment:
I needed to extract this from ```@all_renderable``` in order to call it from the new redirect decorators